django/http/request.py

```diff
@@ -19,11 +19,18 @@
 from django.utils.deprecation import RemovedInDjango40Warning
 from django.utils.encoding import escape_uri_path, iri_to_uri
 from django.utils.functional import cached_property
-from django.utils.http import is_same_domain, limited_parse_qsl
+from django.utils.http import is_same_domain
 from django.utils.regex_helper import _lazy_re_compile
+from django.utils.version import PY38
 
 from .multipartparser import parse_header
 
+if PY38:
```

Comment:
I'd probably just add a comment here with the version support details, so when we come across this again ≈4.x or so we can review.